src/inverseRobot.js

```diff
@@ -1,13 +1,25 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
-
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (robot.hasOwnProperty('name') && robot.name !== 'name') {
+      inversedRobot[robot.name] = 'name';
+    }
```

Comment:
why the `name` is hardcoded here? `robot` can have value with any naming. so just check `key` and `value` from line 6 and 7
```suggestion
    if (robot.hasOwnProperty(value)) {
      inversedRobot[value] = key;
    }
```

Reply:
because here I tried to make property name always come first, for some reason it didnt go as first property and i failed all tests